.github/workflows/scripts/istioCrypto.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+
+# Istio, crypto deployment
+
+export MESH_NAME='Istio'
+export SERVICE_MESH='ISTIO'
+
+# Check if mesheryctl is present, else install it
+if ! [ -x "$(command -v mesheryctl)" ]; then
+    echo 'mesheryctl is not installed. Installing mesheryctl client... Standby... (Starting Meshery as well...)' >&2
+    curl -L https://meshery.io/install | ADAPTERS=istio PLATFORM=kubernetes bash -
+fi
+
+sleep 200
+kubectl get pods -n meshery
+
+echo "Meshery has been installed."
+# mesheryctl system login --provider None
+
+# mesheryctl mesh deploy istio --watch
+
+sleep 200
+
+kubectl get all -n istio-system
+
+# so that istio operator gets initialized
+mesheryctl system stop
+
+sleep 400
+
+mesheryctl system start
+
+# Applying/deploying crpyto pattern
+mesheryctl pattern apply -f ./AadiCrypto.yaml
+
+sleep 200
+
+
+kubectl get all -n istio-operator
+
+# deplyoing httbin application
+mesheryctl app onboard -f  ./httbin.yaml -s "Kubernetes Manifest"
```

Comment:
I think it's `httpbin.yaml`, right? And will it be exposed to outside from ingressgateway?

Reply:
Yes @gyohuangxin the httpbin.yaml file does have the gateway and virtual service configuration